package exceptionHandeling;

import java.util.Iterator;

public class ExceptionPravtice {

	public static void main(String[] args) {

		int y = 10;

		try {
			System.out.println(y / 0);
		} catch (ArithmeticException e) {

		}
		System.out.println(
				"<this is the second time I am telling you to fix your math, I don't wanna tell you all the time > "
						+ "");

		
		
	try {
		
	} catch (Exception e) {
		
	}
	}

}
